backend/wmg/data/query.py

```diff
@@ -0,0 +1,87 @@
+from typing import List
+
+import tiledb
+from pandas import DataFrame
+from pydantic import BaseModel, Field
+from tiledb import Array
+
+UNFILTERED_SLICE = slice(None)
+
+
+class WmgQueryCriteria(BaseModel):
+    gene_ontology_term_ids: List[str] = Field(default=[], unique_items=True, min_items=0)
+    organism_ontology_term_id: str  # required!
+    tissue_ontology_term_ids: List[str] = Field(unique_items=True, min_items=1)  # required!
+    dataset_ids: List[str] = Field(default=[], unique_items=True, min_items=0)
+    assay_ontology_term_ids: List[str] = Field(default=[], unique_items=True, min_items=0)
```

Comment:
fyi The included datasets should only have the following assay_ontology_term_ids
        included_assay_ontology_ids = ['EFO:0008722', 'EFO:0010010', 'EFO:0010550', 'EFO:0010961', 'EFO:0030002', 'EFO:0009901', 'EFO:0011025',  'EFO:0009899', 'EFO:0009900', 'EFO:0009922', 'EFO_0030003', 'EFO:0030004', 'EFO:0008919']
        let me know if it would be helpful for me to store that const somewhere accessible

Reply:
No need; in the next story (in progress now), the API layer will be querying the available ontology term ids (for each dim) directly from the cube; since we need to return refined lists of terms for each query, it makes sense to keep this dynamic.